inf-clojure.el

```diff
@@ -436,7 +436,8 @@ Used by this command to determine defaults."
                                             (file-name-nondirectory file-name)))
   (comint-send-string (inf-clojure-proc)
                       (format inf-clojure-load-command file-name))
-  (inf-clojure-switch-to-repl t))
+  (when current-prefix-arg
```

Comment:
I am using inf-clojure-eval-buffer which has what I wanted. So I do not see the need of modifying inf-clojure-load-file now.  So you can close this as non issue. 

Thanks.